### PR TITLE
VDS-1269: Add createDeployCommit function

### PR DESCRIPTION
**Pull request by mvktsk**
Added ability to commit deployment into the specified branch without PR to support auto-deploy to dev environment for Demo solution

### CODE REVIEW

File: create-deploy-pr/action.yml [hunk 1]
```diff
@@ -29,6 +29,10 @@ inputs:
     description: "Path to config map in repository"
     required: true
     default: "webstore-app/resources/deployment-cm.yaml"
+  forceCommit:
+    description: "Flag to create deploy commit without PR. If true - create commit into deployBranch and don`t create PR"
+    required: true
+    default: "false"
```
Comment (krankenbro) [inline create-deploy-pr/action.yml:35]: Isn't default value redundant? We already defined this parameter as required so it should always get a value.

File: create-deploy-pr/dist/index.js [hunk 2]
```diff
@@ -166,6 +166,48 @@ function createDeployPr(deployData, targetRepo, baseRepo, octokit) {
...
+                            content: Buffer.from(deployContent).toString("base64"),
+                            sha: cmData.sha,
+                            message: "Automated update " + baseRepoName,
+                            committer: {
+                                name: 'vc-ci',
+                                email: 'ci@virtocommerce.com'
+                            },
+                            author: {
+                                name: 'vc-ci',
+                                email: 'ci@virtocommerce.com'
```
Comment (krankenbro) [inline create-deploy-pr/dist/index.js:201]: Shall we move these values to parameters with default values?

File: create-deploy-pr/src/index.ts [hunk 3]
```diff
@@ -119,6 +119,43 @@ async function createDeployPr(deployData: DeploymentData, targetRepo: RepoData,
...
+        content: Buffer.from(deployContent).toString("base64"),
+        sha: cmData.sha,
+        message: `Automated update ${baseRepoName}`,
+        committer:{
+            name: 'vc-ci',
+            email: 'ci@virtocommerce.com' 
+        },
+        author:{
+            name: 'vc-ci',
+            email: 'ci@virtocommerce.com'
```
Comment (krankenbro) [inline create-deploy-pr/src/index.ts:153]: Shall we move these values to parameters with default values?

**Review by krankenbro — APPROVED** [review]

File: create-deploy-pr/action.yml (hunk 1 above)
Comment (mvktsk) [reply create-deploy-pr/action.yml:35]: Default value placed for backward compatibility. In this case, the **required** parameter looks redundant.

File: create-deploy-pr/src/index.ts (hunk 3 above)
Comment (mvktsk) [reply create-deploy-pr/src/index.ts:153]: Good idea!

**Pull request merged**